### PR TITLE
engine: add release notes for 20.10.22

### PR DESCRIPTION
**Pull request by thaJeztah**

### CODE REVIEW

File: engine/release-notes/index.md [hunk 1]
```diff
@@ -15,6 +15,36 @@ for Docker Engine.
...
+- Update Docker Compose to [v2.14.1](https://github.com/docker/compose/releases/tag/v2.14.1){:target="_blank" rel="noopener"}.
+- Update Docker Scan to [v0.23.0](https://github.com/docker/scan-cli-plugin/releases/tag/v0.23.0){:target="_blank" rel="noopener"}.
+- Update containerd (`containerd.io` package) to [v1.6.13](https://github.com/containerd/containerd/releases/tag/v1.6.13){:target="_blank" rel="noopener"},
+  includes fixes prior (1.6.9, 1.6.10, 1.6.11, and 1.6.12) patch releases, which
+  include a fix for [CVE-2022-23471](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2022-23471){:target="_blank" rel="noopener"}.
+- Update Go runtime to [1.18.9](https://go.dev/doc/devel/release#go1.18.minor){:target="_blank" rel="noopener"},
+  includes updates from prior (1.18.8 and 1.18.9) releases, which include fixes for
+  [CVE-2022-41716](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2022-41716){:target="_blank" rel="noopener"},
+  [CVE-2022-41717](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2022-41717){:target="_blank" rel="noopener"}, and
+  [CVE-2022-41720](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2022-41720){:target="_blank" rel="noopener"}.
```
Comment (thaJeztah) [inline engine/release-notes/index.md:35]: I was struggling a bit to pick the right wording; the TL;DR is that there were multiple updates of these between our releases (well, we did publish packages for each of these containerd releases); some of the releases "in between" contained security fixes, and my intent here was to make sure that information doesn't get lost.

File: engine/release-notes/index.md [hunk 2]
```diff
@@ -15,6 +15,36 @@ for Docker Engine.
...
+- Update bundled BuildKit version to to fix `output clipped, log limit 1MiB reached` errors [moby/moby#44339](https://github.com/moby/moby/pull/44339){:target="_blank" rel="noopener"}.
```
Comment (thaJeztah) [inline engine/release-notes/index.md:36]: Copy/pasta: I need to remove this one.

File: engine/release-notes/index.md [hunk 3]
```diff
@@ -15,6 +15,35 @@ for Docker Engine.
...
+
+### Updates
+
+- Update Docker Compose to [v2.14.1](https://github.com/docker/compose/releases/tag/v2.14.1){:target="_blank" rel="noopener"}.
+- Update Docker Scan to [v0.23.0](https://github.com/docker/scan-cli-plugin/releases/tag/v0.23.0){:target="_blank" rel="noopener"}.
+- Update containerd (`containerd.io` package) to [v1.6.13](https://github.com/containerd/containerd/releases/tag/v1.6.13){:target="_blank" rel="noopener"},
+  includes fixes prior (1.6.9, 1.6.10, 1.6.11, and 1.6.12) patch releases, which
+  include a fix for [CVE-2022-23471](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2022-23471){:target="_blank" rel="noopener"}.
+- Update Go runtime to [1.18.9](https://go.dev/doc/devel/release#go1.18.minor){:target="_blank" rel="noopener"},
+  includes updates from prior (1.18.8 and 1.18.9) releases, which include fixes for
```
Comment (dvdksn) [inline engine/release-notes/index.md:32]: Can we remove the middle bit? Makes it easier to read, and I don't think we lose any relevant details.

```suggestion
  which includes fixes for
```

**Review by dvdksn — DISMISSED** [review]

**Review by dvdksn — APPROVED** [review]

File: engine/release-notes/index.md (hunk 3 above)
Comment (thaJeztah) [reply engine/release-notes/index.md]: Ah, yes, that would work; I changed it to "to include fixes for" (for both Go and containerd); it felt less ambiguous (i.e., the latest version itself doesn't have the fix, but "update to `<latest version>` to include fixes for `<vulns>`" doesn't try to implicate that the latest version has the actual fix.

Happy to change to "which" though!

**Pull request merged**